README.md

```diff
@@ -304,12 +307,14 @@ services:
 <br/>
 
 <details><summary>✏ <b>Code Examples</b></summary>
-//docker-compose file
-```
-// Put teardown code that shows how we check for CI
+
+```javascript
+  if (isCI) {
```

Comment:
Give more context - Put comments above, "//global-teardown.js - Where we clean-up after all tests", show jest config like you did before